### PR TITLE
ignore generated man files

### PR DESCRIPTION
**Pull request by Mashimiao**
Signed-off-by: Ma Shimiao <mashimiao.fnst@cn.fujitsu.com>

### CODE REVIEW

**Comment by cyphar** [comment]
LGTM

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,6 @@
 /oci-create-runtime-bundle
 /oci-unpack
 /oci-image-validate
+
+*.1
+*.patch
```
Comment (wking) [inline .gitignore]: Who makes these `*.patch` files?

Comment (Mashimiao) [inline .gitignore]: we always work with patch, and easy to forget to remove them before adding. I think it'd better to ignore them.

Comment (wking) [inline .gitignore]: I use `*.patch` files in projects where contribution is by mailing patches to a mailing list.  But I personally don't use them much in push-based projects like those on GitHub.  If you generate `*.patch` files as part of your personal workflow with this project, I think you should add a local ignore rule (gitignore(5) gives `$HOME/.config/git/ignore` and `$GIT_DIR/info/exclude` as possible locations).  But if rules in the committed `.gitignore` are based on workflow idiosyncrasies, it becomes hard to maintain them.

Comment (Mashimiao) [inline .gitignore]: Fine, updated

**Comment by wking** [comment]
1434e4c looks good to me.

**Comment by caniszczyk** [comment]
@cyphar btw PullApprove should catch your LGTMs moving forward, it was a misconfiguration on my end

**Comment by cyphar** [comment]
LGTM

**Comment by Mashimiao** [comment]
@opencontainers/image-tools-maintainers  PTAL

**Comment by coolljt0725** [comment]
LGTM

**Pull request merged**